Source/NSScrubberItemView.m

```diff
@@ -0,0 +1,34 @@
+/* Implementation of class NSScrubberItemView
+   Copyright (C) 2019 Free Software Foundation, Inc.
```

Comment:
Date mismatch

Reply:
The wrong year is not just here. It is wrong in most of the files. The idea of a code review is not that I have to point out every single instance where the generator resulted in incorrect values. You know better than me where your used the generator. This specific issue is not that important, I think we just ignore it.

Reply:
Yes, I know.  I apologize, I have made the corrections to the generator.   Ill correct the dates ASAP.